site/en/docs/devtools/javascript/breakpoints/index.md

```diff
@@ -157,27 +211,28 @@ all mouse events.
 3.  Check one of these categories to pause whenever any event from that category is fired, or expand
     the category and check a specific event.
 
-{% Img src="image/admin/c40PakNjXuOPIn21VySn.png", alt="Creating an event listener breakpoint.", width="800", height="543" %}
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/aFFgOPi3outuMb32WO3k.png", alt="Creating an event listener breakpoint.", width="800", height="726" %}
 
-**Figure 7**: Creating an event listener breakpoint for `deviceorientation`
+This example shows how to create an event listener breakpoint for `deviceorientation`.
 
 ## Exception breakpoints {: #exceptions }
 
 Use exception breakpoints when you want to pause on the line of code that's throwing a caught or
-uncaught exception.
+uncaught exception. You can pause on both such exceptions independently.
```

Comment:
This is actually currently only true for non-node debug sessions. The reason is that for node LTS, which still has an outdated version of v8, we cannot yet support that. We're planning to make these independent switches available as soon as node LTE will switch versions, and have an updated v8. See crbug: https://crbug.com/1382762

The RFC's FAQ shows how it looks like for Node applications right now: https://github.com/ChromeDevTools/rfcs/discussions/3

Maybe this is worth mentioning?

Reply:
Added a note about it. Thanks!